docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
+- Users with student or non-institutional email address cannot create new projects
```

Comment:
Not currently the case.  We let anybody apply and be assessed.  In mind here, often we see postgrads or even occasionally final-year undergrads requesting access.  ON a successful application, the applying user becomes the Technical Lead.

Reply:
I agree, not convinced of the value given that the approval process is a manual step anyway. At Swansea we also have the luxury of the ability to distinguish between pre-PhD and post-PhD users by email, and we don't permit per-PhD applications historically, but this would only be possible I expect for Swansea users. I expect this would be a "maybe/someday" change, and implemented by someone at Swansea for Swansea, if it ever was implemented.